components/IngredientDetails.js

```diff
@@ -43,8 +60,16 @@ const IngredientDetails = ({ ingredients }) => {
               #{ingredient.flavor}
             </Flavor>
           </CardFooter>
+          <DeleteButton onClick={handleDelete}>Delete</DeleteButton>
         </DetailsCard>
       </Container>
+      {showConfirm && (
+        <ConfirmationDialog>
+          <p>Are you sure you want to delete this ingredient?</p>
+          <button onClick={confirmDelete}>Yes</button>
+          <button onClick={cancelDelete}>No</button>
```

Comment:
adding the type to a button is always helpful:

```suggestion
          <button type="button" onClick={confirmDelete}>Yes</button>
          <button type="button" onClick={cancelDelete}>No</button>
```